int8 path_size
nav_msgs/Path global_path
---
bool result
